cmd/nvkind/cluster_create.go

```diff
@@ -135,6 +135,14 @@ func runClusterCreate(c *cli.Context, f *ClusterCreateFlags) error {
 		}
 	}
 
+	if err := cluster.PrintClusterInfo(); err != nil {
+		return fmt.Errorf("getting cluster info: %w", err)
+	}
+
+	if err := cluster.RegisterNvidiaRuntimeClass(); err != nil {
+		return fmt.Errorf("registering runtime class: %w", err)
+	}
+
```

Comment:
Called these here because I vaguely remember that Evan and I talked about doing this after doing per-node modifications (the loop above this). However, this might be a brain fart.

Reply:
Makes sense. Before the runtime is configured on the nodes, using the runtime class declared in this spec would be  an error.